saleor/graphql/payment/mutations.py

```diff
@@ -603,7 +611,22 @@ class TransactionUpdateInput(graphene.InputObjectType):
     type = graphene.String(
         description="Payment type used for this transaction.",
     )
-    reference = graphene.String(description="Reference of the transaction.")
+    reference = graphene.String(
+        description=(
+            "Reference of the transaction. "
+            "The reference and PSP reference must be unique across all "
+            "`transactionItem` objects. "
+            "DEPRECATED: this field will be removed in Saleor 3.9 (Feature Preview). "
```

Comment:
IMO, you should use `DEPRECATED_IN_3X_INPUT` label to make it consistent.

Reply:
But `DEPRECATED_IN_3X_INPUT` says it will be removed in 4.0, but this is a preview feature, so we don't need to wait till 4.0. We want to give users enough time to switch to new field and drop it in future releases of 3.X